src/harness/certs/README.md

```diff
@@ -109,6 +109,9 @@ Required certificates are:
 * `client_inapplicable.[cert|key]`: leaf CBSD device inapplicable fields certificate
   Used on security test test_WINNF_FT_S_SCS_15.
 
+* `short_lived_client.[cert|key]`: Leaf CBSD certificate that will expire within 5 minutes of creation
```

Comment:
No longer accurate.

Reply:
Removed   this file from the PR, as the short lived certificates for devices and domain   proxy are created at run time and they are no longer needed to be generated   using the "generate_fake_certs.sh" script before the tests are run.